ember-headlessui/addon/components/menu.ts

```diff
@@ -1,19 +1,54 @@
+/* eslint-disable padding-line-between-statements */
 import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
 import { guidFor } from '@ember/object/internals';
 
 import { restartableTask, timeout } from 'ember-concurrency';
 
-export default class Menu extends Component {
+import type Button from './menu/button';
+import type Item from './menu/item';
+import type Items from './menu/items';
+
+interface MenuSignature {
+  Args: {
```

Comment:
are all of these public API?
seems like many are private and shouldn't be exposed

Reply:
thanks, good catch, these types were actually meant to be on the button component. 

I have removed these now. If you find others, I would be happy to know.